index.html

```diff
@@ -0,0 +1,154 @@
+            working as a Digital Marketing Assistant at Favorite Food Ltd.
+          </p>
+          <h2>Skills:</h2>
+          <p>HTML | CSS | JavaScript</p>
+        </div>
+      </section>
+
+      <!-- projects -->
+
+      <section id="projects">
+        <h2>Projects</h2>
+
+        <!-- main project with the clikable link -->
+
+        <div class="project-container-main">
+          <div class="project-image">
+            <a
+              target="_blank"
+              href="https://sumaiasorna.github.io/semantic-html-code-refactor/"
+              ><img
+                src="./assets/images/Horiseon-marketing-website.png"
+                alt="Marketing webpage image"
+            /></a>
+          </div>
+          <div class="project-description">
+            <h3>Refactor Code</h3>
+            <p>
+              Refactoring Horiseon-marketing-website with semantic html for
+              improved accessibility.
+            </p>
+          </div>
+        </div>
+        <!-- six projects in a flex container-->
+        <div class="cards-container">
+          <div class="small-project-card">
+            href="">
+            <img
+              src="./assets/images/project 2.jpg"
+              alt="placeholder project image"
+            />
+
+            <h3>Project 2</h3>
+            <p>coming soon</p>
+          </div>
+          <div class="small-project-card">
+            <img
+              src="./assets/images/project-3 .jpg"
+              alt="placeholder project image"
+            />
+            <h3>Project 3</h3>
+            <p>coming soon</p>
+          </div>
+          <div class="small-project-card">
+            <img
+              src="./assets/images/project-4.jpg"
+              alt="placeholder project image"
+            />
+            <h3>Project 4</h3>
+            <p>coming soon</p>
+          </div>
+          <div class="small-project-card">
+            <img
+              src="./assets/images/project 5.jpg"
+              alt="placeholder project image"
+            />
+            <h3>Project 5</h3>
+            <p>coming soon</p>
+          </div>
+          <div class="small-project-card">
+            <img
+              src="./assets/images/project-6.jpg"
+              alt="placeholder project image"
+            />
+            <h3>Project 6</h3>
+            <p>coming soon</p>
+          </div>
+          <div class="small-project-card">
+            <img
+              src="./assets/images/project-7.jpg"
+              alt="placeholder project image"
+            />
+            <h3>Project 7</h3>
+            <p>coming soon</p>
+          </div>
+        </div>
+      </section>
+    </main>
+    <!-- footer section  and added id for the contact me section -->
+    <footer id="Contact-me">
+      <h3>Sum<span>aia</span> Salah<span>uddin</span></h3>
+      <div class="contact-me">
+        <a href="https://www.linkedin.com/in/sumaia-salahuddin-519521218/"
```

Comment:
Possibly add target="_blank" to these links so they open in a new tab

Reply:
Done.